CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ PROJECT( fc )
 set( CMAKE_CXX_STANDARD 14 )
 SET( CMAKE_CXX_STANDARD_REQUIRED ON )
 
-if( "${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU" )
+if( "${CMAKE_CXX_COMPILER_ID}" MATCHES "^(Apple)?Clang|GNU$" )
```

Comment:
```
CMake Warning (dev) in CMakeLists.txt:
  A logical block opening on the line

    /home/jmjatlanta/Development/cpp/bitshares-fc-1584/CMakeLists.txt:10 (if)

  closes on the line

    /home/jmjatlanta/Development/cpp/bitshares-fc-1584/CMakeLists.txt:14 (endif)

  with mis-matching arguments.
```

Perhaps C-n-P the "if" into the "else" and "endif". Tested with cmake 3.15.1.

Reply:
Done, also in core